frontend/src/components/common/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Did you look into the material library that was mentioned? I thought we were going to use that as a base.

Reply:
I did look into the Material library, but didn't see any components in that library that would fit the progress bar. 

I have installed and tested it in a different branch, but it is not part of this one as it was not required here.